components/Ui/UiPageHeaderWithCard.vue

```diff
@@ -0,0 +1,121 @@
+<template>
```

Comment:
I don't see where this entire component is being used, do we need it?

Reply:
I ran the program with and without the component, and everything looks the same, so I think I'll remove it.